app/components/donation/credit-card.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+const { Component } = Ember;
+
+export default Component.extend({
+  classNames: ['credit-card-donation']
```

Comment:
Wondering if this could be higher level, like just `credit-card-form` or something?